### ISSUE TITLE
Fix issues with accessibility

### ISSUE THREAD

**Issue by epennifold** [exeGesIS-SDM/esdm-mojowebsite-skin#20]
We need our websites to be WCAG AA compliant.

We need to review our main ESDM site and our documentation sites.

This site (https://www.w3.org/WAI/ER/tools/) says that there’s this webtool: “A-Tester checks the pre-enhanced version of a web page designed with progressive enhancement against Evaluera's "WCAG 2.0 Level-AA conformance statements for HTML5 foundation markup" making a report that can serve as a broad and easily confirmed WCAG 2.0 Level-AA claim, even for enhanced versions.”

They’re talking about this tool https://webaccessibility.com/ which says our main page in 87% compliant. So I think if we can fix the main issues, we should be able to say we’re WCAG 2.0 AA compliant.
